devfiles/hosted-che-dogfooding.devfile.yaml

```diff
@@ -58,12 +55,21 @@ components:
         attributes:
           protocol: http
           public: 'true'
-    memoryLimit: 3Gi
+    memoryLimit: 4Gi
     env:
       - name: BUILD_PATH
         value: /tmp
 
 commands:
+
+  - name: set theia to the pinned revision
```

Comment:
Let's name it `1. Set theia to the pinned revision` and shift index of other commands. Just to be consistent with other command and keep the order of commands.

Reply:
Is it better to use `git checkout ${THEIA_COMMIT_SHA}`  ??
`git reset` don't switch to the commit on my side. I just opened a terminal and used `git status` to check.

Reply:
it does check where master is pointing

Reply:
> It would be nice to rename `hosted-che-dogfooding.devfile.yaml` to `workspaces.openshift.com.devfile.yaml`. It will be clear for developers to determine which file need to use for which infrastructure.

Yeah, but eventually i will remove all the devfiles and keep a single one.

Reply:
` ${THEIA_COMMIT_SHA}` is not a branch btw

Reply:
We use `git checkout` when building the images
https://github.com/eclipse-che/che-theia/blob/main/dockerfiles/theia/docker/alpine/builder-clone-theia.dockerfile#L3

Reply:
ok but a dev environment is different from a build environment. If you do any changes in theia, what would be the branch used by default. it should be `master` by default so `git reset --hard` is to be used.

Reply:
Anyway it looks much better if `git status` says me where I am, e.g.
```
$ git status
HEAD detached at 4a627bb
```
It looks nice as well if I changed some files
```
$ git status
HEAD detached at 4a627bb
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   devfile.yaml

no changes added to commit (use "git add" and/or "git commit -a")
```